src/VisualStudio/Core/Test/Snippets/SnippetTestState.vb

```diff
@@ -37,6 +38,9 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Snippets
 
             Workspace.Options = Workspace.Options.WithChangedOption(InternalFeatureOnOffOptions.Snippets, True)
 
+            Dim featureServiceFactory = GetExportedValue(Of IFeatureServiceFactory)()
+            featureServiceFactory.GlobalFeatureService.Disable(PredefinedEditorFeatureNames.AsyncCompletion, EmptyFeatureController.Instance)
```

Comment:
>featureServiceFactory.GlobalFeatureService.Disable(PredefinedEditorFeatureNames.AsyncCompletion, EmptyFeatureController.Instance) [](start = 12, length = 129)

Does this mean we will be running SnippetTests for the old completion only?

Reply:
We are today only running SnippetTests for legacy completion. This change just makes it obvious. 😄